cg/utils/files.py

```diff
@@ -27,3 +30,14 @@ def get_files_matching_pattern(directory: Path, pattern: str) -> List[Path]:
 def get_all_files_in_dir(base_path: Path) -> Set[Path]:
     """Get a set of all files relative to the given base path."""
     return {file.relative_to(base_path) for file in base_path.rglob("*") if file.is_file()}
+
+
+def rename_file(file_path: Path, renamed_file_path: Path) -> None:
+    """Rename the given file path."""
+    file_path.rename(renamed_file_path)
+    LOG.debug(f"Renamed {file_path} to {renamed_file_path}.")
+
+
+def is_pattern_in_file_path(file_path: Path, pattern: str) -> bool:
```

Comment:
If I just saw this function name, I would assume that it checks whether the parent directories contain the pattern as well. Not just the actual file name. Minor detail though, feel free to skip

Reply:
You are right, will make it more explicit.